internal/page/attorney/register.go

```diff
@@ -186,6 +186,10 @@ func makeAttorneyHandle(mux *http.ServeMux, store SessionStore, errorHandler pag
 				return
 			}
 
+			if !page.AttorneyCanGoTo(attorney, r.URL.String()) {
+				http.Redirect(w, r, appData.Lang.URL(page.Paths.Attorney.TaskList.Format(attorney.LpaID)), http.StatusFound)
```

Comment:
`page.Paths.Attorney.TaskList.Redirect(w, r, appData, attorney.LpaID)` is neater for these, also you'll need to `return` after them or it runs through and produces an error